### DISCUSSION TITLE
Welcome to react-native-fbsdk-next Discussions!

### DISCUSSION THREAD

**Discussion by thebergamo** [thebergamo/react-native-fbsdk-next#167] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikehardy** [comment]
Hey :wave: - I used to be a java e-commerce consultant in the 2000s/2010s then I "retired" and now I'm fooling around with mobile and multi-platform technology for fun. More than anything, I just like making computers do what I want, and making software work.

I maintain react-native-firebase, and we use react-native-fbsdk-next as our recommended auth provider for facebook, so I help out here to make sure this library works, and of course I also use it in my personal projects. Cheers

**Comment by iddar** [comment]
Help me! I'm looking a mobile dev with experience in appsflayer and Facebook sdk for react native (temporal or permanent position on my startup)
